yarn.lock

```diff
@@ -7796,14 +7814,14 @@ metro-react-native-babel-transformer@0.66.0, metro-react-native-babel-transforme
     metro-source-map "0.66.0"
     nullthrows "^1.1.1"
 
-metro-resolver@0.66.0, metro-resolver@^0.64.0, metro-resolver@^0.66.0:
+metro-resolver@0.66.0, metro-resolver@^0.66.0:
   version "0.66.0"
   resolved "https://registry.yarnpkg.com/metro-resolver/-/metro-resolver-0.66.0.tgz#068c1bb98cd80c239f051e7b0f43f00e763b889d"
   integrity sha512-JUbkmznwgMSb5ViFpvau5sPD7uuZekToVsitrDLTAeMPruvjxjae2+XSIai9NmcGSprfvqyGYURlz0qXu1YnJQ==
   dependencies:
     absolute-path "^0.0.0"
 
-metro-runtime@0.66.0, metro-runtime@^0.64.0, metro-runtime@^0.66.0:
+metro-runtime@0.66.0, metro-runtime@^0.66.0:
```

Comment:
No need to fix, just sharing observation:
* Double-Entry of 0.66.0 in the key.

Reply:
Yeah, I noticed that. I think metro and react-native differently declare the semver range